app/controllers/v0/branches_of_service_controller.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+module V0
+  class BranchesOfServiceController < PreneedsController
+    # We call authenticate_token because auth is optional on this endpoint.
+    skip_before_action(:authenticate)
```

Comment:
additionally can we move this and all other controllers into its own namespace / module? We're adding 5-6 new controllers and I think it will clean things up a bit if we namespace.